source/aks/patching-artifactory.html.md.erb

```diff
@@ -1,17 +1,17 @@
 ---
 title: Artifactory Patching
 weight: 30
-last_reviewed_on: 2024-03-13
+last_reviewed_on: 2024-06-04
 review_in: 12 months
 ---
 
 # <%= current_page.data.title %>
 
-Artifactory is utilized in the context of managing and updating software dependencies and configurations within a Kubernetes environment. Here's how it is used based on the provided instructions:
+Artifactory is utilized in the context of managing and updating software dependencies and configurations within a Kubernetes environment. "Here's" how it is used based on the provided instructions:
 
-The steps below will show you how to check the current version of Artifactory being used (7.71.5) and then to update it to a newer version (7.77.3). This process involves accessing Artifactory's website to determine the latest version available.
+The steps below will show you how to check the current version of Artifactory being used (7.77.3) and then to update it to a newer version (7.84.12). This process involves accessing ["Artifactory's website"] (https://jfrog.com/help/r/jfrog-release-information/artifactory-release-notes) to determine the latest version available.
 
-Before applying the update, a testing process is initiated. This involves downloading a repository containing configuration files for Artifactory (artifactory-sbox.yaml). This file is created and modified to reflect the desired changes, such as updating the version number.
+Before applying the update, a testing process is initiated. This involves downloading the repository containing configuration files for Artifactory and creating (artifactory-sbox.yaml). This file is then modified to reflect the desired changes, such as updating the version number.
```

Comment:
Worth including an example PR of how you done this in Flux? (Kustomization etc)

Reply:
I think it would be good, when I ran through the process it was confusing so adding in the pr will help add context.